testing/test_assertion.py

```diff
@@ -772,56 +772,70 @@ def test_mojibake(self) -> None:
 
 class TestAssert_reprcompare_dataclass:
     @pytest.mark.skipif(sys.version_info < (3, 7), reason="Dataclasses in Python3.7+")
-    def test_dataclasses(self, testdir):
+    def test_dataclasses(self, testdir, consecutive=True):
```

Comment:
Note that `consecutive=True` should be passed as a keyword argument to `fnmatch_lines`; the above has no effect.

Reply:
get it, thank you, tried to check if I did it right, but didn't understand it.